03_letters/CameronSchafer/letters.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// string to rune with counts
+func letters(s string) map[rune]int {
+	tMap := make(map[rune]int)
+
+	for i := range s {
+		tMap[rune(s[i])]++
+	}
```

Comment:
You could consider??:

```
for _, r := range s {
    tMap[r]++
}
```

Reply:
I changed it to this and it looks much better! thank you 👍